lib/main.js

```diff
@@ -299,13 +316,40 @@ let load = loader({
     }
   },
 
+  apiClient: {
+    requires: ['cfg'], 
+    setup: async({cfg}) => {
+      let http = require('http');
+      let taskcluster = require('taskcluster-client');
+      let baseUrl = cfg.server.baseUrl;
+      let reference = api.reference({baseUrl});
+      let ClientClass = taskcluster.createClient(reference);
+      return new ClientClass({
+        agent: http.globalAgent,
+        baseUrl,
+        credentials: cfg.taskcluster.credentials,
+      });
+
+    },
+  },
+
+  purgequeues: {
+    requires: ['server', 'apiClient'],
+    setup: async ({server, apiClient}) => {
+      await apiClient.purgeQueues();
```

Comment:
Am I reading this correctly that we're calling a method on ourselves via http? Why not do this work directly?